exercises/04-deploy-to-cf.md

```diff
@@ -48,7 +48,12 @@ Run the following to build and package your application:
 npm run ci-build && npm run ci-package
 ```
 
-Then push it to Cloud Foundry:
+Then deploy the data defined in the CSVs to the HDI container:
```

Comment:
How is this deploying anything to the HDI container? We ran this in the getting started section. The documentation there states, that this deploys the database locally. How can it now do something else?

If I went through this sample I would not understand the difference between what we did in the getting started section and here.

Reply:
Edit: I now scrolled further down and saw, that you changed the `npm cds-deploy` script. Consider putting this into the manifest.yml or alternatively create two scripts, one for local, one for productive deployment (e. g. `npm cds-deploy:local` and `...:prod`). Don't forget to mention what the difference is.